SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -345,7 +345,7 @@ void USpatialPlayerSpawner::ReceiveForwardedPlayerSpawnRequest(const Worker_Comm
 	{
 		bool bUnresolvedRef = false;
 		AActor* PlayerStart = Cast<AActor>(FUnrealObjectRef::ToObjectPtr(PlayerStartRef, NetDriver->PackageMap, bUnresolvedRef));
-		bRequestHandledSuccessfully = !bUnresolvedRef;
+		bRequestHandledSuccessfully = PlayerStart != nullptr;
```

Comment:
In this instance, bUnresolvedRef was false even though the PlayerStart was unresolved?

Reply:
Yeah, ToObjectPtr does not always fill in this value (only if the reference is in the packagemap). I toyed with fixing this function instead but it's used in a lot of places, it seems kind of troll for a return value to sometimes be set. 

It's only set and true when nullptr is returned and the ref is in the packagemap, thoughts?